scripts/populate_mock_db.py

```
@@ -0,0 +1,71 @@
+
+    c.executemany(
+        "INSERT INTO crew (name, role, skills, contract_hours) VALUES (?, ?, ?, ?)",
+        crew_data,
+    )
+
+    # Create appliances
+    c.execute("INSERT INTO appliance (name) VALUES (?)", ("P22P6",))
+    c.execute("INSERT INTO appliance (name) VALUES (?)", ("P22M1",))
+
+    # Create availability
+    now = datetime.now()
```

Ruff (0.15.0)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)